terraform/modules/signon_oauth_application/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "additional_tags" {
+  default     = {}
+  description = "Additional resource tags"
+  type        = map(string)
+}
+
+variable "app_name" {
+  type        = string
+  description = "Name for a Signon OAuth application resource, e.g. publishing_api"
+}
+
+variable "app_shortname" {
+  type = string
+}
+
+variable "description" {
+  type = string
+}
+
+variable "environment" {
+  type = string
+}
+
+variable "home_uri" {
+  type = string
+}
+
+variable "permissions" {
+  type = list(string)
+}
+
+variable "redirect_path" {
+  type = string
+}
+
+variable "workspace" {
```

Comment:
Is there a reason we're passing workspace around as a var? Is it not the same as terraform.workspace?

Reply:
`local.workspace` replaces ["default" with "ecs"](https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/govuk-publishing-platform/defaults.tf#L2).

Reply:
Ah ok, thanks.